src/Application.ts

```diff
@@ -0,0 +1,32 @@
+import { ExpressServer } from './ExpressServer';
+import { GithubExtractor } from './GithubExtractor';
+import { Logger } from './Logger';
+
+export class Application {
+
+    constructor(private readonly timeBetweenExtraction: number, private readonly logger: Logger, private readonly githubExtractor: GithubExtractor, private readonly expressServer: ExpressServer) {
```

Comment:
On essaie de chop down les paramètres de ctor. Ça aide la lecture des dépendances et allège les CR quand il y a des changements.

```suggestion
    constructor(
        private readonly timeBetweenExtraction: number,
        private readonly logger: Logger,
        private readonly githubExtractor: GithubExtractor,
        private readonly expressServer: ExpressServer,
    ) {
```